### DISCUSSION TITLE
How to resolve this error?

### DISCUSSION THREAD

**Discussion by feynmans-socks** [ArthurSonzogni/FTXUI#255] [Q&A] [answered]
Visual Studio 2019 

LNK2019	unresolved external symbol "class std::shared_ptr<class ftxui::Node> __cdecl ftxui::operator|(class std::shared_ptr<class ftxui::Node>,class std::function<class std::shared_ptr<class ftxui::Node> __cdecl(class std::shared_ptr<class ftxui::Node>)>)" (??Uftxui@@YA?AV?$shared_ptr@VNode@ftxui@@@std@@V12@V?$function@$$A6A?AV?$shared_ptr@VNode@ftxui@@@std@@V12@@Z@2@@Z) referenced in function "public: __thiscall <lambda_d45c10ce5a431827cd79926d163c102c>::operator()(void)const " (??R<lambda_d45c10ce5a431827cd79926d163c102c>@@QBE@XZ)

This error occurs when I try to build any of the examples

**Comment by ArthurSonzogni** [comment, marked as answer]
I guess this was incorrectly linking against ftxui.

There is a starter project using cmake here:
https://github.com/ArthurSonzogni/ftxui-starter
As an example.
